modules/single_page_checkout/reg_steps/attendee_information/attendee_info_single.template.php

```diff
@@ -25,31 +25,32 @@
 $prev_ticket = 0;
 
 if (count($registrations) > 0) {
+    $ticketID = key($template_args['ticket_count']);
     ?>
 
-    <div id="spco-attendee-panel-dv-single"
-        class="spco-attendee-panel-dv spco-attendee-ticket-single">
+    <div id="spco-attendee-panel-dv-<?php echo $ticketID; ?>"
+        class="spco-attendee-panel-dv spco-attendee-ticket-<?php echo $ticketID; ?>">
 
         <div class="spco-ticket-info-dv small-text">
             <?php if (!is_admin()) : ?>
-                <h5><?php _e('Details', 'event_espresso'); ?></h5>
+                <h5><?php esc_html_e('Details', 'event_espresso'); ?></h5>
```

Comment:
thnx 👍

Reply:
sorry this wasn't actually part of this PR again! But I'll accept your goodwill regardless.